kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -270,13 +270,6 @@ define([
                     break;
 
                 // JOB messages
-                case 'job_canceled':
-                    this.sendBusMessage(JOB, msgData.job_id, 'job-canceled', {
-                        jobId: msgData.job_id,
-                        via: 'job_canceled',
-                    });
-                    break;
```

Comment:
There is no message of type `job-canceled` that gets sent by the backend

Reply:
🤦